diff -up tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Alone/Makefile.orig tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Alone/Makefile
--- tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Alone/Makefile.orig	2019-01-11 21:12:33.328905173 -0800
+++ tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Alone/Makefile	2019-01-11 21:12:50.269004003 -0800
@@ -3,7 +3,7 @@ CXX = g++ -O2 -Wall
 CXX_C = gcc -O2 -Wall
 LIB = -lm
 RM = rm -f
-CFLAGS = -c -I ../../../
+CFLAGS = -c -I ../../../ -fpermissive
 
 OBJS = \
   LzmaAlone.o \
diff -up tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Lib/makefile.orig tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Lib/makefile
--- tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Lib/makefile.orig	2019-01-11 21:17:20.138573991 -0800
+++ tools/squashfs-tools/lzma/SRC/7zip/Compress/LZMA_Lib/makefile	2019-01-11 21:17:31.034637209 -0800
@@ -2,7 +2,7 @@ PROG = liblzma.a
 CXX = g++ -O3 -Wall
 AR = ar
 RM = rm -f
-CFLAGS = -c -I../../..
+CFLAGS = -c -I../../.. -fpermissive
 
 OBJS = \
   ZLib.o \
