diff -up progs.gpl/pppd.alpha/Makefile.orig progs.gpl/pppd.alpha/Makefile
--- progs.gpl/pppd.alpha/Makefile.orig	2019-01-11 20:10:36.024261196 -0800
+++ progs.gpl/pppd.alpha/Makefile	2019-01-11 20:45:53.312088965 -0800
@@ -118,10 +118,11 @@ CFLAGS   += -DHAS_SHADOW
 #LIBS     += -lshadow $(LIBS)
 endif
 
-ifneq ($(wildcard /usr/include/crypt.h),)
+# we shouldn't be checking /usr for anything, we have our own local toolchain
+ifneq ($(shell find ../.. -name crypt.h),)
 CFLAGS   += -DHAVE_CRYPT_H=1
 endif
-ifneq ($(wildcard /usr/lib/libcrypt.*),)
+ifneq ($(shell find ../.. -name libcrypt.*),)
 LIBS	+= -lcrypt
 endif
 
